lib/main/tsconfig/tsconfig.ts

```diff
@@ -234,6 +240,9 @@ export var defaults: ts.CompilerOptions = {
     target: ts.ScriptTarget.ES5,
     module: ts.ModuleKind.CommonJS,
     moduleResolution: ts.ModuleResolutionKind.NodeJs,
+    baseUrl: undefined,
+    paths: undefined,
+    rootDirs: undefined,
```

Comment:
No need to provide defaults if they are not activated (basically let the TS compiler do its own default thing). Also explicit undefined initialization not needed (it `undefined` on read anyways) :rose: